code/src/main/java/com/onlinecontacttracing/storage/data-storage/TimeInterval.java

```diff
@@ -20,6 +20,15 @@ public TimeInterval(long intervalStart, long intervalEnd) {
     intervalEndSeconds = intervalEnd;
   }
 
+  public String displayTimeIntervalAsDate(int timeZoneOffsetMinutes) {
+    return String.format("%s to %s", convertToDate(intervalStartSeconds, timeZoneOffsetMinutes), convertToDate(intervalEndSeconds, timeZoneOffsetMinutes));
+  }
+
+  private static String convertToDate(long timeInSeconds, int timeZoneOffsetMinutes) {
+    long offsetTimeInMillis = (timeInSeconds - timeZoneOffsetMinutes * 60) * 1000;
```

Comment:
See coment above. Using a TimeZone Object we won't have to do this kind of math.